### PR TITLE
Closure Test

### PR DESCRIPTION
**Pull request by bamesserly**

### CODE REVIEW

File: includes/CVUniverse.cxx [hunk 1]
```diff
@@ -849,8 +851,8 @@ double CVUniverse::GetWeight() const {
   wgt_rpa = GetRPAWeight();
 
   // MINOS efficiency
-  if (!m_is_truth && GetBool("isMinosMatchTrack"))
-    wgt_mueff = GetMinosEfficiencyWeight();
+//  if (!m_is_truth && GetBool("isMinosMatchTrack"))
```
Comment (bamesserly) [inline includes/CVUniverse.cxx:854]: We obviously don't want to remove these weights on the master branch. This was just done for the closure.

@egranadosvazquez do you remember if we must remove these lines for the closure test? Or instead did we just remove them to be safe?

Comment (bamesserly) [reply includes/CVUniverse.cxx:854]: These weights or any of the others.

File: loadLibs.C [hunk 2]
```diff
@@ -34,6 +35,26 @@ void loadLibs(bool verbose_cvu = true) {
...
+  // Add GXSE to the inlude path
+  {
+    gInterpreter->AddIncludePath( gSystem->ExpandPathName("$GENIEXSECEXTRACTROOT") );
+    std::string newpath = std::string(gROOT->GetMacroPath()) + ":" + std::string(gSystem->ExpandPathName("$GENIEXSECEXTRACTROOT"));
+    gROOT->SetMacroPath( newpath.c_str() );
+    gSystem->Load( gSystem->ExpandPathName("$GENIEXSECEXTRACTROOT/libGENIEXSecExtract.so") );
+  }
+
+  {
+    string newpath = string(gROOT->GetMacroPath()) + ":" + string("${PLOTUTILSROOT}/../bin" );
```
Comment (bamesserly) [inline loadLibs.C:47]: what are these lines about? Did I add these??

**Comment by egranadosvazquez** [comment]
I didn't observe changes when I commented those lines

**Comment by egranadosvazquez** [comment]
yes, you added it

**Pull request merged**